crucible-llvm/crucible-llvm.cabal

```diff
@@ -67,6 +67,8 @@ library
     Lang.Crucible.LLVM.Extension
     Lang.Crucible.LLVM.Globals
     Lang.Crucible.LLVM.Intrinsics
+    Lang.Crucible.LLVM.Intrinsics.Libc
+    Lang.Crucible.LLVM.Intrinsics.LLVM
```

Comment:
These modules are now exported to support registering overrides without needing to provide a list of `L.Decl`s, which aren't available in some settings, i.e., in `crucible-llvm-cli`.